src/sliding-tab/ExNavigationSlidingTab.js

```diff
@@ -205,8 +207,8 @@ class ExNavigationSlidingTab extends PureComponent<any, Props, State> {
     }
   };
 
-  _renderHeader = (props) => {
-    const TabBarComponent = this.props.position === 'top' ? TabBarTop : TabBar;
+  _renderTabBar = (props) => {
+    const TabBarComponent = this.props.position === 'top' || 'bottom' ? TabBarTop : TabBar;
```

Comment:
There is actually an error here, this basically translates to,

```js
(this.props.position === 'top') || 'bottom' ? TabBarTop : TabBar;
```

Which is not what you want. Anyways, I'd suggest leaving this change out